nimble/host/mesh/src/cfg_cli.c

```diff
@@ -815,17 +815,13 @@ static int node_identity_status(struct bt_mesh_model *model,
 	}
 
 	status = net_buf_simple_pull_u8(buf);
-	net_idx = net_buf_simple_pull_le16(buf) & 0xfff;
+	net_buf_simple_pull_le16(buf);
```

Comment:
would be good to explain in commit message why this action is not necessary

Reply:
done